dojson/contrib/to_marc21/fields/bd80x83x.py

```diff
@@ -332,7 +332,5 @@ def reverse_series_added_entry_uniform_title(self, key, value):
         '8': utils.reverse_force_list(
             value.get('field_link_and_sequence_number')),
         '$ind1': '_',
-        '$ind2': value.get(
-            'nonfiling_characters',
-            '_'),
+        '$ind2': value.get('nonfiling_characters') if value.get('nonfiling_characters') in valid_nonfiling_characters else '_',
```

Comment:
We have JSONSchema that one should follow. If there are not correct data we will generate incorrect output better than hiding it.